### PR TITLE
readme installation guide

### PR DESCRIPTION
**Pull request by sowasred2012**
## Done

Updated readme with Ubuntu installation guide

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,133 @@
...
+
+Open two terminals in the Indico virtualenv:
+```
+cd ~/dev/indico
+source ./env/bin/activate
+```
+
+In one terminal run the webpack watcher:
+```
+./bin/maintenance/build-assets.py indico --dev --watch
```
Comment (carkod) [inline README.md:107]: is this inside of the `./env/bin` ? I can't get past this point

Comment (petesfrench) [reply README.md:107]: This command should be ran from the indico project folder that contains the `env` folder. This should be the same folder where the `src` is kept.

Comment (carkod) [reply README.md:107]: The issue was that I was in the wrong folder. So maybe we should add
```
cd ~/dev/indico/src
```

**Comment by marinacastejon** [comment]
@carkod can you please take a look at this? thanks

**Review by carkod — APPROVED** [review]
LGTM

**Pull request merged**